reward/reward_distributor_test.go

```diff
@@ -17,15 +17,104 @@
 package reward
 
 import (
+	"fmt"
 	"math/big"
 	"testing"
 
 	"github.com/klaytn/klaytn/blockchain/types"
 	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/consensus/istanbul"
 	"github.com/klaytn/klaytn/params"
 	"github.com/stretchr/testify/assert"
 )
 
+type testGovernance struct {
+	p *params.GovParamSet
+}
+
+func newTestGovernance(intMap map[int]interface{}) *testGovernance {
+	p, _ := params.NewGovParamSetIntMap(intMap)
+	return &testGovernance{p}
+}
+
+func newDefaultTestGovernance() *testGovernance {
+	return newTestGovernance(map[int]interface{}{
+		params.Epoch:               604800,
+		params.Policy:              params.WeightedRandom,
+		params.UnitPrice:           25000000000,
+		params.MintingAmount:       "9600000000000000000",
+		params.Ratio:               "34/54/12",
+		params.UseGiniCoeff:        true,
+		params.DeferredTxFee:       true,
+		params.MinimumStake:        "5000000",
+		params.StakeUpdateInterval: 86400,
+	})
+}
+
+func (governance *testGovernance) Params() *params.GovParamSet {
+	return governance.p
+}
+
+func (governance *testGovernance) ParamsAt(num uint64) (*params.GovParamSet, error) {
+	return governance.p, nil
+}
+
+func (governance *testGovernance) setTestGovernance(intMap map[int]interface{}) {
+	p, _ := params.NewGovParamSetIntMap(intMap)
+	governance.p = p
+}
+
+var (
+	cnBaseAddr     = 500
+	stakeBaseAddr  = 600
+	rewardBaseAddr = 700
+	minStaking     = uint64(2000000) // changing this value will not change the governance's min staking
+	minted, _      = big.NewInt(0).SetString("9600000000000000000", 10)
+	proposer       = common.StringToAddress("0x1552F52D459B713E0C4558e66C8c773a75615FA8")
+	kir            = intToAddress(1000)
+	kgf            = intToAddress(2000)
```

Comment:
```suggestion
	proposerAddr       = common.StringToAddress("0x1552F52D459B713E0C4558e66C8c773a75615FA8")
	kirAddr            = intToAddress(1000)
	kgfAddr            = intToAddress(2000)
```

variables `proposer`, `kir`, `kgf` are reward amounts in reward_distributor.go. It's confusing to use the names.

Reply:
fixed in 62457e30